spec/models/package_manager/npm_spec.rb

```diff
@@ -55,12 +55,12 @@
       end
     end
 
-    context "latest version is deprecated" do
```

Comment:
this existing spec was labeled "is deprecated" but was actually identical to the "isn't deprecated" spec above, so I'm replacing it with my new spec